lib/config/config.go

```diff
@@ -107,6 +107,8 @@ func New(myID protocol.DeviceID) Configuration {
 	util.SetDefaults(&cfg.Options)
 	util.SetDefaults(&cfg.GUI)
 
+	cfg.Options.UnackedNotificationIDs = []string{"authenticationUserAndPassword"}
```

Comment:
I think this fits best in `.prepare`.

Reply:
Fixed as suggested

Reply:
I really don't know what I was thinking when suggesting this: It means the notification is readded on every change to the folder - that's bad. I am sorry for the back-and-forth...

Reply:
No problem, will it be ok if I will go back to the version when it is added in `.New`?

Reply:
Sure. And just to avoid a misunderstanding: The other change you made in that commit in `clean` was good.